### PR TITLE
replaced entires to entries

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #6622, I replaced  **entires**  to **entries**  in row line no 116

### CODE REVIEW

File: windows/client-management/troubleshoot-inaccessible-boot-device.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ To verify the BCD entries:
 2. In the **Windows Boot Loader**  that has the **{default}** identifier, make sure that **device** , **path** , **osdevice,**  and **systemroot**  point to the correct device or partition, winload file, OS partition or device, and OS folder.
  
    >[!NOTE]
-   >If the computer is UEFI-based, the **bootmgr**  and **winload**  entires under **{default}**  will contain an **.efi**  extension.
+   >If the computer is UEFI-based, the **bootmgr**  and **winload**  entries under **{default}**  will contain an **.efi**  extension.
```
Comment (illfated) [inline windows/client-management/troubleshoot-inaccessible-boot-device.md:116]: The correction looks good. I would like to use the opportunity to request a MarkDown indent marker compatibility spacing and simplify some double spacing in the sentence:
```suggestion
   > If the computer is UEFI-based, the **bootmgr** and **winload** entries under **{default}** will contain an **.efi** extension.
```

File: windows/client-management/troubleshoot-inaccessible-boot-device.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ To verify the BCD entries:
 2. In the **Windows Boot Loader**  that has the **{default}** identifier, make sure that **device** , **path** , **osdevice,**  and **systemroot**  point to the correct device or partition, winload file, OS partition or device, and OS folder.
  
    >[!NOTE]
```
Comment (illfated) [inline windows/client-management/troubleshoot-inaccessible-boot-device.md:115]: Also, please add this MarkDown indent marker compatibility spacing (even if it is not within your change):
```suggestion
   > [!NOTE]
```

**Comment by RAJU2529** [comment]
@illfated . Ok sir carry on in your freetime

**Comment by illfated** [comment]
#sign-off

**Comment by denisebmsft** [comment]
@RAJU2529 and @illfated thank you

**Pull request merged**